### ISSUE TITLE
What a PITA 3.6.8 was for me - glad to be back on 3.6.7

### ISSUE THREAD

**Issue by chaloots** [QL-Win/QuickLook#657]
At first all seemed well, as I was glad to see such a recent update to an app I've grown quite fond of and reliant on. The first signs of trouble came when it kept loading previously viewed docs and images instead of the ones selected. This eventually evolved into merely loading thumbnails. I tried to find a way to uninstall it; no luck, as 3.6.8 didn't appear in Geek uninstaller, the uninstall feature of CCleaner (3.6.7 was visible in both) or in Windows Control Panel. I used WizFile to find any and all QuickLook entries, but as there were so many due to the multiple versions I didn't know what to delete. Add to all this confusion was my forgetting which particular flavor of installer worked best (.msi vs .exe). As a final resort I downloaded 3.6.7 zip and used its' .exe to go back to 3.6.7

Windows 10, version 1903

Reliability Monitor report - QuickLook 

Summary: Stopped working

Date ‎5/‎21/‎2020 08:48 AM

Status Report sent

Description
Faulting Application Path:	C:\Users\___\Documents\W10 downloads\W10 installed\QuickLook v3.6.8\QuickLook v3.6.8.exe

Problem signature
Problem Event Name:	APPCRASH
Application Name:	QuickLook v3.6.8.exe
Application Version:	3.6.8.0
Application Timestamp:	5ec03270
Fault Module Name:	QuickLook.Native64.dll
Fault Module Version:	0.0.0.0
Fault Module Timestamp:	5ec03268
Exception Code:	80000003
Exception Offset:	0000000000008b55
OS Version:	10.0.18362.2.0.0.256.48
Locale ID:	1033
Additional Information 1:	0ee9
Additional Information 2:	0ee9746d30e9f23e7310b9ceb3433d51
Additional Information 3:	ef17
Additional Information 4:	ef17b553fb24976df66459349acc41d4

Extra information about the problem
Bucket ID:	ab044f68d0020b22176845207c62a19f (1686674066278359455)

Successful application reconfiguration 5/19/2020 07:30 PM:
Windows Installer installed the product. Product Name: QuickLook. Product Version: 3.6.8.0. Product Language: 1033. Manufacturer: Paddy Xu. Installation success or error status: 

Unsuccessful application reconfiguration 5/20/2020 05:22 PM:
Windows Installer reconfigured the product. Product Name: QuickLook. Product Version: 3.6.8.0. Product Language: 1033. Manufacturer: Paddy Xu. Reconfiguration success or error status: 1603.

========5/20/2020 05:18:13 PM========
System.InvalidOperationException: Cannot determine compressed stream type. Supported Archive Formats: Zip, GZip, Tar, Rar, 7Zip, LZip
   at SharpCompress.Archives.ArchiveFactory.Open(Stream stream, ReaderOptions readerOptions)
   at QuickLook.Plugin.ArchiveViewer.ArchiveInfoPanel.LoadItemsFromArchive(String path)
   at QuickLook.Plugin.ArchiveViewer.ArchiveInfoPanel.<>c__DisplayClass13_0.<BeginLoadArchive>b__0()

========4/26/2020 09:28:38 AM========
System.Runtime.InteropServices.COMException (0x80010001): Call was rejected by callee. (Exception from HRESULT: 0x80010001 (RPC_E_CALL_REJECTED))
   at QuickLook.Plugin.OfficeViewer.IInitializeWithFile.Initialize(String pszFilePath, UInt32 grfMode)
   at QuickLook.Plugin.OfficeViewer.PreviewHandlerHost.Open(String path)
   at QuickLook.Plugin.OfficeViewer.Plugin.View(String path, ContextObject context)
   at QuickLook.ViewerWindow.<>c__DisplayClass5_0.<BeginShow>b__0()

**Comment by xupefei** [comment]
Apologies for the negative experience.

> The first signs of trouble came when it kept loading previously viewed docs and images instead of the ones selected

Could you elaborate?

**Comment by chaloots** [comment]
Loading a doc in 3.6.8 was sluggish, but they eventually appeared in QL. After that, all subsequent selections showed the most previous view. They too were slow in loading, not nearly as quickly as QL had in past versions. For instance, in an alphabetical sorting, selecting doc "A" and invoking QL, and waiting, would ultimately be revealed. Selecting "B" would be sluggish and then preview doc "A". So on for "C", "D", etc.

The next attempt at explanation will be difficult as I now have QL 3.6.7 installed, and this QL version (3.6.7) behaves as it did in versions prior to 3.6.8 so I cannot provide even an accurate screenshot (and do NOT wish to go back to 3.6.8 to prove a point), but in the past, when invoking QL on a doc which QL does not natively support, there would be a thumbnail rectangle image representation of the file. This was most evident when attempting to use QL on Microsoft Office docs (prior to my installing the appropriate plugin), where pertinent info regarding the doc was made visible (title, file format), but no searching or scrolling was enabled.

**Comment by xupefei** [comment]
Thanks for the reply! I have no Office installed so I can't try it on my PC -- but I have another bug in 3.6.8 which is (I believe) related to your issue. Can you try the latest version 3.6.9?

**Comment by chaloots** [comment]
As a long time mac user I'd come to rely heavily on QuickLook in that platform. It and Preview are my main reasons for continued use of macOS. When I discovered QL for Windows I was excited to try it. There have been some issues in the past, mainly with it mysteriously quitting but as I'm ignorant to coding or manipulating/tweaking programs under the hood I was reluctant to post my difficulties here. I perceive Github to be a trusted, reputable source for software.

As previously mentioned, QL has become a most favored personal app. With that and my regard for the developer in mind, I am, against my better judgement, going to install 3.6.9

It's been my experience in the past, even after ensuring that "unblock" was checked in properties, of being unable to find an .exe file once having opened QL in zip form. With that in mind I've downloaded the .msi version (am I the only one who has had similar difficulties in discovering the more recognizable "exe" anywhere in the un-zipped folder?).
